core/services/ocr2/plugins/mercury/plugin.go

```diff
@@ -180,10 +180,22 @@ type factoryCfg struct {
 	feedID                utils.FeedID
 }
 
+func getPluginFeedIDs(pluginConfig config.PluginConfig) (linkFeedID utils.FeedID, nativeFeedID utils.FeedID) {
+	if pluginConfig.LinkFeedID != nil {
+		linkFeedID = *pluginConfig.LinkFeedID
+	}
+	if pluginConfig.NativeFeedID != nil {
+		nativeFeedID = *pluginConfig.NativeFeedID
+	}
+	return linkFeedID, nativeFeedID
```

Comment:
If the plugin config is missing either of the Link or Native feed ID, this will return the default un-initialised value for the `FeedID` type ([32]byte). 

I'm not sure if this will cause issues down-stream, but it might be useful to at least emit a warning that these values are un-set.

Reply:
I would actually prefer not to warn on this, since it's expected for all exclusively keystone workflow-based jobs. Also, the Feed IDs are validated if the pluginConfig is present at all (`len > 0`) so users will receive an error if they try to set a job spec with an incomplete pluginConfig.